src/hooks/useProvidersList.js

```diff
@@ -271,23 +278,30 @@ const useProvidersList = (options, updatedAt = null) => {
   }, [filters]);
 
   const loadProviders = () => {
-    if (storeSession.tradeApi.accessToken) {
+    if (storeSession.tradeApi.accessToken && connectedProviders) {
       /**
-       * @type {ProvidersPayload}
+       * @type {NewAPIProvidersPayload}
        */
-      const payload = {
-        token: storeSession.tradeApi.accessToken,
-        type: connectedOnly ? "connected" : "all",
-        ro: true,
-        provType: provType,
+      const payload2 = {
+        type,
         timeFrame,
         internalExchangeId,
       };
 
       tradeApi
-        .providersGet(payload)
+        .providersGetNew(payload2)
         .then((responseData) => {
-          const uniqueProviders = uniqBy(responseData, "id");
+          let uniqueProviders = uniqBy(responseData, "id");
+          uniqueProviders.forEach((provider, index) => {
```

Comment:
It would have been nice to use `map` here to return the original/modified provider.

Reply:
already done